c_glib/Brewfile

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 brew "autoconf-archive"
-brew "glib-utils"
+brew "glib"
```

Comment:
Can we remove this line too? I think that `glib` is installed automatically as a dependency of `gobject-introspection`.

Reply:
Done! Thanks @kou !